data/fields/barrier.json

```diff
@@ -1,5 +1,33 @@
 {
     "key": "barrier",
     "type": "typeCombo",
-    "label": "Type"
+    "label": "Type",
+     "strings": {
+         "options": {
+             "fence": "Fence",
+             "gate": "Gate",
+             "wall": "Wall",
+             "hedge": "Hedge",
+             "kerb": "Kerb",
+             "bollard": "Bollard",
+             "lift_gate": "Lift Gate",
```

Comment:
Note that "lift gate", term used in OSM tagging is not used in actual English. 

It seems to be referred to as a boom gate / boom barrier.

https://en.wikipedia.org/wiki/Boom_barrier

Reply:
Fixed